src/main/java/com/connectCo/domain/coupon/coupon.java

```diff
@@ -0,0 +1,46 @@
+package com.connectCo.domain.coupon;
+
+import com.connectCo.domain.BaseEntity.baseEntity;
+import com.connectCo.domain.enums.CouponType;
+import com.connectCo.domain.store.store;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+public class coupon extends baseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
```suggestion
    @GeneratedValue(generator = "uuid2")
    @GenericGenerator(name = "uuid2", strategy = "uuid2")
    @Column(columnDefinition = "BINARY(16)")
```
UUID로 할 경우 이렇게 선언해주는게 좋아